src/app/project-overview/component.ts

```diff
@@ -200,6 +213,32 @@ export class ProjectOverviewComponent implements OnInit, OnDestroy {
       .subscribe();
   }
 
+  private _loadClusterMachineDeployments(cluster: Cluster): void {
```

Comment:
I think both Overview and List page has same method? can we keep it one and move it under MD service file?